README.md

```diff
@@ -375,7 +375,10 @@ dataset_shard_idx:
 sequence_len: 2048
 # max sequence length to concatenate training samples together up to
 # inspired by StackLLaMA. see https://huggingface.co/blog/stackllama#supervised-fine-tuning
+# soon to be DEPRECATED
```

Comment:
```suggestion
# FutureWarning: This will soon to be DEPRECATED
```

Also, what would be the con of leaving this alone, but just warn that there is a better option?